fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -511,6 +554,79 @@ public void addQuantityValues(List<IDatabaseObject> group, String prefix) {
         model.addTable(tbl);
     }
 
+    /**
+     * Add the COMPOSITES table for the given resource name prefix
+     * <pre>
+  parameter_name_id       INT  NOT NULL,
+  comp1_str            BIGINT,
+  comp1_number         BIGINT,
+  comp1_date           BIGINT,
+  comp1_token          BIGINT,
+  comp1_quantity       BIGINT,
+  comp1_latlng         BIGINT,
+  ...
+  logical_resource_id  BIGINT  NOT NULL
+
+CREATE INDEX idx_device_composites_pttr ON device_composites(parameter_name_id, comp1_token, comp2_token, resource_id);
+CREATE INDEX idx_device_composites_ptqr ON device_composites(parameter_name_id, comp1_token, comp2_quantity, resource_id);
+CREATE INDEX idx_device_composites_rptt ON device_composites(resource_id, parameter_name_id, comp1_token, comp2_token);
+CREATE INDEX idx_device_composites_rptq ON device_composites(resource_id, parameter_name_id, comp1_token, comp2_quantity);
+ALTER TABLE device_composites ADD CONSTRAINT fk_device_composites_pnid FOREIGN KEY (comp1_str)      REFERENCES device_str_values;
+ALTER TABLE device_composites ADD CONSTRAINT fk_device_composites_rid  FOREIGN KEY (comp1_number)   REFERENCES device_number_values;
+ALTER TABLE device_composites ADD CONSTRAINT fk_device_composites_rid  FOREIGN KEY (comp1_date)     REFERENCES device_date_values;
+ALTER TABLE device_composites ADD CONSTRAINT fk_device_composites_rid  FOREIGN KEY (comp1_token)    REFERENCES device_token_values;
+ALTER TABLE device_composites ADD CONSTRAINT fk_device_composites_rid  FOREIGN KEY (comp1_quantity) REFERENCES device_quantity_values;
+ALTER TABLE device_composites ADD CONSTRAINT fk_device_composites_rid  FOREIGN KEY (comp1_latlng)   REFERENCES device_latlng_values;
+...
+ALTER TABLE device_quantity_values ADD CONSTRAINT fk_device_quantity_values_pn FOREIGN KEY (parameter_name_id) REFERENCES parameter_names ON DELETE CASCADE;
+ALTER TABLE device_quantity_values ADD CONSTRAINT fk_device_quantity_values_r  FOREIGN KEY (resource_id)       REFERENCES device_logical_resources ON DELETE CASCADE;
+     * </pre>
+     * @param prefix
```

Comment:
missing the param group

Reply:
it turns out the javadoc for this parameter was missing form every other method in this class as well...will add it everywhere

Reply:
done